cmd/notation/policy/import.go

```diff
@@ -42,7 +42,12 @@ func importCmd() *cobra.Command {
 Example - Import trust policy configuration from a file:
   notation policy import my_policy.json
 `,
-		Args: cobra.ExactArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("requires 1 argument but received %d.\nUsage: notation policy import <path-to-policy.json>\nPlease specify a trust policy file as the argument", len(args))
+			}
+			return nil
```

Comment:
Do we have E2E test regarding this change? If not, please add it.

Reply:
Updated.